packages/material-ui-styles/src/createStyles/createStyles.d.ts

```diff
@@ -8,6 +8,10 @@ import { StyleRules } from '@material-ui/styles/withStyles';
  * @param styles a set of style mappings
  * @returns the same styles that were passed in
  */
-export default function createStyles<C extends string, P extends object>(
+// For TypeScript v3.5 P has to extend {} instead of object
+// See https://github.com/mui-org/material-ui/issues/15942
+// and the breaking changes section here
+// https://devblogs.microsoft.com/typescript/announcing-typescript-3-5/
```

Comment:
Fix seems find though I'm still fairly certain this issue is not what's described in the breaking changes section (though it might be caused by it).

Could you link to https://github.com/microsoft/TypeScript/issues/31735 instead so that we know if the the breaking changes actually imply this changed behavior or if this was an unrelated breaking change.

Reply:
Yeah, seems strange.
Link updated